### PR TITLE
Ticket6678 razorbill rp100

### PR DESCRIPTION
**Pull request by davidkeymer**
Support module

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+iocBoot_DEPEND_DIRS += $(filter %App,$(DIRS))
+
+# Add any additional dependency rules here:
+
+include $(TOP)/configure/RULES_TOP
+

+ioctests:

+	.\system_tests\run_tests.bat
```
Comment (Tom-Willemsen) [inline Makefile]: Some nasty characters at end-of-lines here which I think causes build trouble - I think this is fixed in latest version of IOC gen but can you fix here please?

File: rzbrp100Sup/rzbrp100_channel.template [hunk 2]
```diff
@@ -0,0 +1,152 @@
+
+#   Output
+
+record(bi, "$(P)CH$(CHANNEL):OUTPUT")
+{
+    field(DESC, "Channel $(CHANNEL) Output Status")
+    field(DTYP, "stream")
+    field(INP, "@rzbrp100.proto getOutput($(CHANNEL)) $(PORT)")
+    field(SCAN, "2 second")
```
Comment (Tom-Willemsen) [inline rzbrp100Sup/rzbrp100_channel.template]: Bit slower than usual - is the device unable to cope with a higher poll rate? If so can you mention that on the wiki?

File: rzbrp100Sup/rzbrp100_channel.template [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+    field(OUT,  "@rzbrp100.proto setOutput($(CHANNEL)) $(PORT)")
+
+    field(ZNAM, "OFF")
+    field(ONAM, "ON")
+
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:CH$(CHANNEL):OUTPUT:SP PP")
+    field(SDIS, "$(P)DISABLE")
+
+    info(INTEREST, "HIGH")
```
Comment (Tom-Willemsen) [inline rzbrp100Sup/rzbrp100_channel.template]: SP not usually interesting as blocks should almost never be on an SP directly.

File: system_tests/lewis_emulators/Rzbrp100/interfaces/stream_interface.py [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+@has_log
+class Rzbrp100StreamInterface(StreamInterface):
+
+    in_terminator = "\r\n"
+    out_terminator = "\r\n"
+
+    commands = {
+        
+        # Catch-all command for debugging
+        CmdBuilder("catch_all").arg("^#9.*$").eos().build(),
```
Comment (Tom-Willemsen) [inline system_tests/lewis_emulators/Rzbrp100/interfaces/stream_interface.py]: This will hide errors - should be removed.

File: Makefile (hunk 1 above)
Comment (davidkeymer) [reply Makefile]: Characters are `<CR>`.  Now replaced with `<CR><LF>`.

Comment (davidkeymer) [reply Makefile]: Also replaced `<TAB>` with four spaces for consistency.

File: rzbrp100Sup/rzbrp100_channel.template (hunk 3 above)
Comment (davidkeymer) [reply rzbrp100Sup/rzbrp100_channel.template]: Have reduced `INTEREST` level to `LOW` for setpoints.

File: rzbrp100Sup/rzbrp100_channel.template (hunk 2 above)
Comment (davidkeymer) [reply rzbrp100Sup/rzbrp100_channel.template]: Have reduced scan period to `1 second` for all relevant records.

File: system_tests/lewis_emulators/Rzbrp100/interfaces/stream_interface.py (hunk 4 above)
Comment (davidkeymer) [reply system_tests/lewis_emulators/Rzbrp100/interfaces/stream_interface.py]: Have removed requested lines.

File: Makefile (hunk 1 above)
Comment (davidkeymer) [reply Makefile]: Replaced spaces with `<TAB>` to enable successful build.

**Review by Tom-Willemsen — APPROVED** [review]

**Pull request merged**